packages/shared/src/consts/providers/configurations/provider-configuration.ts

```diff
@@ -165,6 +163,27 @@ export const fcmGroupConfigurations: ConfigConfigurationGroup[] = [
     enabler: 'inboundWebhookEnabled',
     setupWebhookUrlGuide: 'https://developer.android.com/develop/ui/views/notifications/build-notification',
   },
+  {
+    groupType: 'crossChannelConfigs',
+    configurations: [
+      {
+        key: 'inboxCount',
+        displayName: 'Include in-app count',
```

Comment:
```suggestion
        displayName: 'Use inbox count in badge',
```

Reply:
@scopsy the "badge" is one of the potential use cases, that's why I added a more generic message

Reply:
@scopsy If you want to make the count more visible to users, you might render the custom notification layout and show the number on it.